roles/openshift_logging_fluentd/templates/2.x/fluentd.j2

```diff
@@ -61,7 +61,7 @@ spec:
         - name: varlog
           mountPath: /var/log
         - name: varlibdockercontainers
-          mountPath: /var/lib/docker/containers
+          mountPath: /var/lib/docker
           readOnly: true
```

Comment:
Does fluentd pod works with either /var/lib/docker or /var/lib/docker/containers without any modification? I would expect these 2 directories to contain different content.